### PR TITLE
fix: avoid nesting <p> tags; use default font stack for ecosystem help

### PR DESCRIPTION
**Pull request by paulschreiber**
## Description
avoid nesting <p> tags; use default font stack for ecosystem helper text

### CODE REVIEW

**Comment by paulschreiber** [comment]
@josebui is there w ay to get the default font stack without manually specifying it?

File: src/landscape/components/LandscapeForm/ProfileStep.js [hunk 1]
```diff
@@ -66,10 +66,8 @@ const AreaTypesHelperText = props => {
 
 const EcosystemTypesHelperText = () => {
   return (
-    <Box sx={{ p: 2 }}>
-      <Typography>
-        <Trans i18nKey="landscape.form_profile_ecosystem_types_helper_text" />
-      </Typography>
+    <Box sx={{ p: 2, fontFamily: 'Roboto, Helvetica, Arial, sans-serif' }}>
```
[the remark this answers is not in the document]
Comment (paulschreiber) [reply src/landscape/components/LandscapeForm/ProfileStep.js:69]: How do you do that? I looked in theme.js but didn't see anything obvious.

[the remark this answers is not in the document]
Comment (josebui) [reply src/landscape/components/LandscapeForm/ProfileStep.js:69]: @paulschreiber ohh I missed your question and I asked the same thing, let me see

[the remark this answers is not in the document]
Comment (josebui) [reply src/landscape/components/LandscapeForm/ProfileStep.js:69]: @paulschreiber it seems to be in `theme.typography.fontFamily` See: https://mui.com/material-ui/customization/typography/

Comment (paulschreiber) [inline src/landscape/components/LandscapeForm/ProfileStep.js:69]:
```suggestion
    <Box sx={{ p: 2, fontFamily: theme.typography.fontFamily }}>
```

File: src/landscape/components/LandscapeForm/ProfileStep.js [hunk 2]
```diff
@@ -66,10 +68,8 @@ const AreaTypesHelperText = props => {
 
 const EcosystemTypesHelperText = () => {
   return (
-    <Box sx={{ p: 2 }}>
-      <Typography>
-        <Trans i18nKey="landscape.form_profile_ecosystem_types_helper_text" />
-      </Typography>
+    <Box sx={{ p: 2, fontFamily: theme.typography.fontFamily }}>
```
Comment (josebui) [inline src/landscape/components/LandscapeForm/ProfileStep.js:71]: Suggestion: You don't need to import the theme you have it in side the sx property
```suggestion
    <Box sx={theme => ({ p: 2, fontFamily: theme.typography.fontFamily })}>
```

File: src/landscape/components/LandscapeForm/ProfileStep.js [hunk 3]
```diff
@@ -32,6 +32,8 @@ import { withProps } from 'react-hoc';
 
 import Actions from './Actions';
 
+import theme from 'theme';
+
```
Comment (paulschreiber) [inline src/landscape/components/LandscapeForm/ProfileStep.js:36]:
```suggestion
```

**Review by josebui — APPROVED** [review]

**Pull request merged**